src/styles/rhd-theme/components/_cards.scss

```diff
@@ -287,9 +329,10 @@
 
 .rhd-c-card__title:first-child {
   max-height: 70px;
+  min-height: 70px;
   &:hover {
-    height: auto;
-    min-height: auto;
+    //height: auto;
```

Comment:
I think these can be completely removed, rather than commented out.

Reply:
done